lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -1879,6 +1879,58 @@ LogicalResult ConvertAtenOp<AtenRsubScalarOp>::matchAndRewrite(
   return success();
 }
 
+/// tosa.conv2d does not support group convolution.
+/// Therefore, we have to create multiple ops where the input, kernel
+/// and bias is a slice of the original inputs.
+/// Afterwards we have to concat the results into a single tensor.
+/// This is similar to the legalization done by onnx-mlir.
```

Comment:
@mgehre-amd , this is somewhat based on the `onnx-mlir` repo. Should we include something else for copyrights?